### ISSUE TITLE
Abnormal fetal abdominal circumference

### ISSUE THREAD

**Issue by michellejoywang** [obophenotype/human-phenotype-ontology#11337]
**Preferred term label:**
Abnormal fetal abdominal circumference

**Synonyms**
Abnormal fetal AC, abnormal fetal abdomen

**Definition (free text, please give PubMed ID)**
The fetal abdominal circumference is typically measured in a transverse view using the outer borders of the skin line at the plane where the fetal stomach, umbilical vein and portal sinus is visible. Abnormal fetal abdominal circumference is defined as less than 10%ile or greater than 90%ile for gestational age. 

**Parent term (use hpo.jax.org/app)**
Abnormal fetal gastrointestinal system morphology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-5177-8109

**Comment by michellejoywang** [comment]
AC < 10%ile is also synonym for fetal growth restriction (defined as EFW <10%ile or AC <10%ile)

**Comment by michellejoywang** [comment]
@VarenyaJ PMID and also ?<3%ile (whether or not ?severe growth restriction or sub-set?)

**Comment by pnrobinson** [comment]
I have added this term and also "Increased"/"Decreased" so that it goes into the next release. We may need to revisit the definitions.
